01-node-tutorial/answers/writeWithPromiseAwait.js

```diff
@@ -0,0 +1,31 @@
+const { writeFile, readFile } = require("fs").promises;
+
+const writer = async () => {
+    try {
+        await writeFile('../content/temp.txt', 'I am writing something to my temp.txt file \n', { flag: 'a' });
+        await writeFile('../content/temp.txt', 'I am writing something to my temp.txt file \n', { flag: 'a' });
+        await writeFile('../content/temp.txt', 'I am writing something to my temp.txt file \n', { flag: 'a' });
```

Comment:
Not a bad idea to change the text slightly…but you’ll see the correct number appended so no harm done